### ISSUE TITLE
[Bug Report] reset does not return info

### ISSUE THREAD

**Issue by Kallinteris-Andreas** [Farama-Foundation/PettingZoo#926]
### Describe the bug

`return_info` argument was removed in 1.22.4, but was not made to always return info by default
https://github.com/Farama-Foundation/Gymnasium/commit/3a8daafce113c6161190c0252c6a3a340cc38fd4

### Code example

```shell
from pettingzoo.butterfly import pistonball_v6
env = pistonball_v6.parallel_env(render_mode="human")

observations, info = env.reset() # does not return info
```

### System info

pettingzoo 1.22.4

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/Farama-Foundation/PettingZoo/issues) in the repo

**Comment by Kallinteris-Andreas** [comment]
Update
```
from pettingzoo.butterfly import pistonball_v6
env = pistonball_v6.env()

observations, info = env.reset() # does not return info
```
AEC also does not work

**Comment by elliottower** [comment]
Fixed via #930
